src/Routes.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import Home from "./pages/Home";
+import Favorites from "./pages/Favorites";
+import Navbar from "./components/Navigation/Navbar";
+
+import { Container } from "./App.style";
+import NotFound from "./pages/NotFound";
+
+const Routes = () => {
```

Comment:
In this specific exercise you may not see the value but you are declaring the routes in a semi static way on a separate file. Remember that one of the key takeaways of react router 4 is integrating the routes through the component views dynamically, as your application grows and become more complex maintaining this static definition of routes may not be the best approach